src/components/blocks/wrappers/LocalLanguageAlternatives.tsx

```diff
@@ -117,3 +79,16 @@ export default LocalLanguageAlternatives;
 
 const languageSDKInterfaceClean = (language: string, selectedTab: string) =>
   language.includes(`_`) ? (language.includes(`${selectedTab}_`) ? language.split('_', 2)[1] : '') : language;
+
+/* this method helps us to determine if an array of languages has a pageSelected present, a condition added if it is an SDK Interface, so we can compare the values correctly */
+const checkIfPreHaveLanguageSelected = (
+  languages: string[],
+  selectedPageLanguage: string,
+  selectedSDKInterfaceTab: string,
+  isSDKInterface: boolean,
+) =>
+  languages
+    .map((lang) =>
+      isSDKInterface ? lang === `${selectedSDKInterfaceTab}_${selectedPageLanguage}` : lang === selectedPageLanguage,
+    )
+    .includes(true);
```

Comment:
Awesome! Sorry, I realize the feedback is a bit vague with 'slightly unusual' - what I meant was that it's a (very tiny) bit inefficient to run it through two operations rather than one. I should have said, you could have kept the implementation as you preferred it and it would be fine. It would be faster as a for loop as well, but a reduce keeps it 'more functional' so that's why I suggested that option.

Nice corrections of the mistakes in my suggested code too :D